### DISCUSSION TITLE
Welcome to SFTPGo Discussions!

### DISCUSSION THREAD

**Discussion by drakkan** [drakkan/sftpgo#258] [General]
## :wave: Welcome!

We’re using Discussions as a place to connect with other members of our community. We hope that you:

- Ask questions you’re wondering about.
- Share ideas.
- Engage with other community members.
- Welcome others and are open-minded. Remember that this is a community we build together :muscle:.

**Comment by fdefilippo** [comment]
We need a logo!

![288-2881446_image-result-for-golang-go-programming-language-logo](https://user-images.githubusercontent.com/16714986/103178996-01548300-4888-11eb-9a24-a2999c1d62a7.png)

**Reply by drakkan** [reply]
yes, thank you! We definitely need a logo
